app/services/ClickHandler.coffee

```diff
@@ -0,0 +1,69 @@
+$ = require 'jquery'
+
+class ClickHandler
+
+  constructor: (@board, two, @clicked=[]) ->
```

Comment:
since you're calling ClickHandler only on line 4 of Board.coffee with only the first two parameters, it might be better to put ```@clicked = []``` on the first line of the constructor instead of the parameters. It's just a stylistic thing